core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/STDCMStandardAllowance.kt

```diff
@@ -44,21 +44,23 @@ fun applyAllowance(
     comfort: Comfort?,
     blockAvailability: BlockAvailabilityInterface,
     departureTime: Double,
-    stops: List<TrainStop>
+    stops: List<TrainStop>,
+    isMareco: Boolean = true,
 ): Envelope {
     if (standardAllowance == null
         || standardAllowance.getAllowanceTime(envelope.totalTime, envelope.totalDistance) < 1e-5
```

Comment:
`areTimesEqual()` or use `TIME_EPSILON` ? (or whatever it's called)

Reply:
`areTimesEqual` has been written on the other branch

It's not part of the diff, we can leave it as it is for now